fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/services/AbstractService.java

```diff
@@ -64,7 +64,13 @@ protected Node findNode(final Session session, final String path) {
         }
     }
 
-    private static void tagHierarchyWithPairtreeMixin(final Node baseNode, final Node createdNode)
+    /**
+     *
+     * @param baseNode
```

Comment:
Based on reading these javadocs, I, as a API consumer, have no better understanding of what this method does. 
Please add some detail.